test/test_response.py

```diff
@@ -377,6 +379,73 @@ def test_empty_stream(self):
 
         self.assertRaises(StopIteration, next, stream)
 
+    def test_length_no_header(self):
+        fp = BytesIO(b'12345')
+        resp = HTTPResponse(fp, preload_content=False)
+        assert resp.length_remaining is None
```

Comment:
We don't use pytest in urllib3. Please use the appropriate `assert*` methods, e.g.,

```py
self.assertIsNone(resp.length_remaining)
```